files/image_config/interfaces/interfaces-config.sh

```diff
@@ -34,6 +34,48 @@ for intf_pid in $(ls -1 /var/run/dhclient*.Ethernet*.pid 2> /dev/null); do
     [ -f ${intf_pid} ] && kill `cat ${intf_pid}` && rm -f ${intf_pid}
 done
 
+
+# Setup eth1 if we connect to a remote chassis DB.
+PLATFORM=${PLATFORM:-`sonic-cfggen -H -v DEVICE_METADATA.localhost.platform`}
+PLATFORM_ENV="/usr/share/sonic/device/$PLATFORM/platform_env.conf"
+[ -f $PLATFORM_ENV ] && source $PLATFORM_ENV
+
+ASIC_CONF="/usr/share/sonic/device/$PLATFORM/asic.conf"
+[ -f $ASIC_CONF ] && source $ASIC_CONF
+
+if [[ -n "$MIDPLANE_SUBNET" && ($NUM_ASIC -gt 1) ]]; then
+    for asic_id in `seq 0 $((NUM_ASIC - 1))`; do
+       NET_NS="asic$asic_id"
+
+       PIDS=`ip netns pids "$NET_NS" 2>/dev/null`
+       if [ "$?" -ne "0" ]; then # namespace doesn't exist
+          continue
+       fi
+
+       # Use /16 for loopback interface
+       ip netns exec $NET_NS ip addr add 127.0.0.1/16 dev lo
+       ip netns exec $NET_NS ip addr del 127.0.0.1/8 dev lo
+
+       # Create eth1 in database instance
+       ip link add name ns-eth1 link eth1-midplane type ipvlan mode l2
```

Comment:
Ok so here we assume the eth1-midplane interface already exists by the time this script is run. Also this interface is created by a platform specific script/service,

Reply:
That's correct. Assumption here is that eth1-midplane was created by a sonic-platform-* service.